dns/dns.go

```diff
@@ -25,34 +17,13 @@ type ServiceDnsRecord struct {
 	StackName   string
 }
 
-func init() {
-	var name string
-	name = os.Getenv("ROOT_DOMAIN")
-	if len(name) == 0 {
-		logrus.Fatalf("ROOT_DOMAIN is not set")
-	}
-	TTLEnv := os.Getenv("TTL")
-	i, err := strconv.Atoi(TTLEnv)
-	if err != nil {
-		TTL = 300
-	} else {
-		TTL = i
-	}
-
-	if !strings.HasSuffix(name, ".") {
-		name = name + "."
-	}
-
-	RootDomainName = name
-}
-
 func ConvertToServiceDnsRecord(dnsRecord DnsRecord) ServiceDnsRecord {
 	splitted := strings.Split(dnsRecord.Fqdn, ".")
 	serviceRecord := ServiceDnsRecord{dnsRecord.Fqdn, splitted[0], splitted[1]}
 	return serviceRecord
 }
 
-func ConvertToFqdn(serviceName string, stackName string, environmentName string) string {
-	domainNameEntries := []string{serviceName, stackName, environmentName, RootDomainName}
-	return strings.ToLower(strings.Join(domainNameEntries, "."))
+func ConvertToFqdn(serviceName, stackName, environmentName, rootDomainName string) string {
```

Comment:
thinking we should rename this class to utils, so from now on all dns util functions go there. Thoughts?